django/team_management/urls.py

```diff
@@ -15,16 +15,15 @@
 """
 from django.contrib import admin
 from django.urls import path
-from django.conf import settings
-from django.conf.urls.static import static
+#from django.conf import settings
+#from django.conf.urls.static import static
```

Comment:
No need to comment things out to save for later. If we need to see how something once was, we can go back in git. I'm going to delete and commit these.

Reply:
Yep. I forgot about those.